### PR TITLE
added trigger button function

### PR DESCRIPTION
**Pull request by flighter18**
at the last wedding, the people didn't use the buzzer because the trigger button on the screen was shown. They just used just those html buttons to trigger the camera

now i implemented the trigger button function
you get 3 option in the admin interface.
html, gpip and html + gpio

html:
The html buttons are visible but access key dosen't work

html + gpio:
The html buttons are visible but access key works

gpio:
No html buttons for triggering the camera just access key

### CODE REVIEW

**Comment by andi34** [comment]
Filemode changes make this impossible to merge.

**Comment by flighter18** [comment]
Wie meinst du das genau? :)

**Comment by andi34** [comment]
Chmod ist schuld. Immer ```git config core.fileMode false``` setzen ;)
Screenshot_20190925-100905__01

**Comment by andi34** [comment]
```
git reset --soft HEAD^
git config core.fileMode false
git commit -a
```
Jenachdem ggf alles per mit sudo ausführen.

Wenns dann passt
```
git push https://github.com/flighter18/photobooth HEAD:triggerbutton -f
```

File: lang/en.js [hunk 1]
```diff
@@ -46,7 +46,8 @@ L10N = {
 	"gallery_no_image": "The gallery is still empty. Take some pictures!",
 	"use_mail": "Use E-Mail",
 	"use_mobile_view": "Enable mobile view",
-	"use_gpio_button": "Use Hardware Button to take a Picture",
+	"general_trigger_button": "Select the type of trigger button",
+        "use_button": "Use Buzzer to take a Picture",
```
Comment (andi34) [inline lang/en.js]: Whitespace

File: lang/es.js [hunk 2]
```diff
@@ -46,7 +46,8 @@ L10N = {
 	"gallery_no_image": "La galeri es vacía. ¡Saque fotos!",
 	"use_mail": "Utilizar el correo electrónico",
 	"use_mobile_view": "Active la vista móvil",
-	"use_gpio_button": "Utilice el botón GPIO para sacar una foto",
+	"general_trigger_button": "Seleccione el tipo de botón de activación",
+        "use_button": "Utilice el botón para sacar una foto",
```
Comment (andi34) [inline lang/es.js]: Whitespace

File: lang/fr.js [hunk 3]
```diff
@@ -46,7 +46,8 @@ L10N = {
 	"gallery_no_image": "La galerie est vide. Faites des photos!",
 	"use_mail": "Utiliser le courrier électronique",
 	"use_mobile_view": "Activez la vue mobile",
-	"use_gpio_button": "Utilisez le bouton matériel pour prendre une photo",
+        "general_trigger_button": "Sélectionnez le type de bouton de déclenchement",
```
Comment (andi34) [inline lang/fr.js]: Whitespace

File: index.php [hunk 4]
```diff
@@ -139,8 +161,15 @@
 			<?php if($config['use_qr']){ echo '<a href="#" class="btn qrbtn"><span class="qrbtnlabel"><i class="fa fa-qrcode"></i> <span data-l10n="qr"></span></span></a>'; } ?>
 			<?php if($config['use_mail']){ echo '<a href="#" class="btn mailbtn"><span class="mailbtnlabel"><i class="fa fa-cloud-download"></i> <span data-l10n="mail"></span></span></a>'; } ?>
 			<?php if($config['use_print']){ echo '<a href="#" class="btn printbtn"><i class="fa fa-print"></i> <span data-l10n="print"></span></a>'; } ?>
-			<a href="#" class="btn newpic"><i class="fa fa-camera"></i> <span data-l10n="newPhoto"></span></a>
-			<?php if($config['use_collage']){ echo '<a href="#" class="btn newcollage"><i class="fa fa-camera"></i> <span data-l10n="newCollage"></span></a>'; } ?>
+                      	<?php
```
Comment (andi34) [inline index.php]: One tab to much

File: lang/de.js [hunk 5]
```diff
@@ -46,7 +46,8 @@ L10N = {
 	"gallery_no_image": "Die Galerie ist noch leer. Mache doch ein paar Bilder!",
 	"use_mail": "E-Mail aktivieren",
 	"use_mobile_view": "Mobile Ansicht aktivieren",
-	"use_gpio_button": "Hardware-Taster verwenden, um ein Bild aufzunehmen",
+	"general_trigger_button": "Auslöser Typ wahlen",
+        "use_button": "Buzzer verwenden, um ein Bild aufzunehmen",
```
Comment (andi34) [inline lang/de.js]: Whitespace

**Comment by flighter18** [comment]
> Chmod ist schuld. Immer `git config core.fileMode false` setzen ;)
> Screenshot_20190925-100905__01

Shit :/
Ok danke für den Tipp.

**Comment by flighter18** [comment]
> ```
> git reset --soft HEAD^
> git config core.fileMode false
> git add --all
> git commit -a
> ```
> 
> Jenachdem ggf alles mit sudo ausführen.
> 
> Wenns dann passt
> 
> ```
> git push https://github.com/flighter18/photobooth HEAD:triggerbutton -f
> ```
Sollte nun gut sein. :)

**Comment by andi34** [comment]
Leider nein

**Comment by andi34** [comment]
Ich kann dir das nachher oder morgen spätestens fixen. Muss mich gleich erstmal um Sperrmüll kümmern und danach Zeit mit Frau und Kind verbringen :)

**Comment by andi34** [comment]
https://github.com/andi34/photobooth/commit/5e5d9c676b1389692e4af84ea8c35219b867afcf

**Comment by andi34** [comment]
```
git reset --hard HEAD^
git fetch https://github.com/andi34/photobooth triggerbutton
git cherry-pick 5e5d9c676b1389692e4af84ea8c35219b867afcf
git push https://github.com/flighter18/photobooth HEAD:triggerbutton -f
```

**Comment by andi34** [comment]
@flighter18  I'd like to wait for @sualko to have the responsive Theme ready before this can go in. Maybe he can add it on top of his changes so it'll be in his PR integrated later?

**Comment by sualko** [comment]
If my pr is merged I'm glad to help here.

**Comment by andi34** [comment]
prepared on top of the responsive theme. style changes only need to be moved and css file recreated after that. https://github.com/andi34/photobooth/commits/triggerbutton

**Comment by andi34** [comment]
@flighter18 , @sualko takes care of the changes once he's done with the refactoring. I'll close this PR for now.

**Pull request closed without merging**